class.jetpack.php

```diff
@@ -5824,11 +5825,17 @@ public static function get_jetapck_options_for_reset() {
 
 			// non_compact
 			'activated',
+
+			// private
+			'register',
+			'blog_token',                  // (string) The Client Secret/Blog Token of this site.
+			'user_token',                  // (string) The User Token of this site. (deprecated)
+			'user_tokens'
 		);
 
 		// Remove the whitelisted Jetpack options
```

Comment:
Nitpick, but now that you have renamed them to `unsafe`, this comment is confusing.